app-admin/xstow/xstow-1.1.1.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="Replacement for GNU stow with extensions"
+HOMEPAGE="https://xstow.sourceforge.net/"
+SRC_URI=https://github.com/majorkingleo/xstow/releases/download/${PV}/${P}.tar.bz2
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="ncurses"
+
+DEPEND="ncurses? ( sys-libs/ncurses:= )"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	default
+
+	eautoreconf
```

Comment:
Do we need `inherit autotools` and `eautoreconf` now we're using a dist tarball?